site-src/v1alpha2/guides/http-redirect-rewrite.md

```diff
@@ -0,0 +1,180 @@
+# HTTP path redirects and rewrites
+
+[HTTPRoute resources](/v1alpha2/api-types/httproute) can issue redirects to
+clients or rewrite paths sent upstream using
+[filters](/v1alpha2/api-types/httproute#filters-optional). This guide shows how
+to use these features.
+
+Note that redirect and rewrite filters are mutually incompatible. Rules cannot
+use both filter types at once.
+
+## Redirects
+
+Redirects return HTTP 3XX responses to a client, instructing it to retrive a
+different resource. [`RequestRedirect` rule
+filters](v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.HTTPRequestRedirectFilter)
+instruct Gateways to emit a redirect response to requests matching a filtered
+HTTPRoute rule.
+
+Redirect filters can substitute various URL components independently. For
+example, to issue a permanent redirect (301) from HTTP to HTTPS, configure
+`requestRedirect.statusCode=301` and `requestRedirect.scheme="https"`:
+
+```
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: HTTPRoute
+metadata:
+  name: http-filter-redirect
+spec:
+  hostnames:
+    - redirect.example
+  rules:
+    - filters:
+      - type: RequestRedirect
+	    requestRedirect:
+          scheme: https
+		  statusCode: 301
```

Comment:
Formatting fix needed here.

Reply:
I *think* maybe this is some tabs instead of spaces?

Reply:
Yeah, my editor was doing some :thinking: stuff there. The now separated examples should be using 4 space indents throughout.